mycore-wcms2/src/main/java/org/mycore/wcms2/navigation/MCRWCMSDefaultSectionProvider.java

```diff
@@ -65,12 +63,13 @@ public class MCRWCMSDefaultSectionProvider implements MCRWCMSSectionProvider {
         "optgroup", "option", "textarea", "keygen", "output", "progress", "meter", "details", "summary", "menuitem",
         "menu", "font");
 
-    private List<String> mycoreTagList = new ArrayList<>();
+    private final List<String> tagList = new ArrayList<>();
 
     public MCRWCMSDefaultSectionProvider() {
-        String mycoreTagListString = MCRConfiguration2.getString("MCR.WCMS2.mycoreTagList").orElse("");
-        for (String tag : mycoreTagListString.split(",")) {
-            mycoreTagList.add(tag.trim());
+        tagList.addAll(HTML_TAG_LIST);
+        String configTagListString = MCRConfiguration2.getString("MCR.WCMS2.mycoreTagList").orElse("");
+        for (String tag : configTagListString.split(",")) {
+            tagList.add(tag.trim());
```

Comment:
suggestion use streams:
(look at MCRJerseyUtil, l. 188)
  List<String> applicationPaths = MCRConfiguration2
            .getOrThrow("MCR.Jersey.Resource.ApplicationPaths", MCRConfiguration2::splitValue)
            .collect(Collectors.toList());

Reply:
It's not the same. getOrThrow() throws an exception instead of returning an empty string.